lua/chatcommands/ponyrp.lua

```diff
@@ -0,0 +1,19 @@
+RegisterChatCommand({'ponyrp'}, function(ply, arg)
+	if !Safe(ply) and !ply:InVehicle() then
+		if !IsValid(ply) then return end
+		ply:SetPos(Vector(-388, 1400, -118)) --treatment room location
+
+		for k, v in pairs(ents.GetAll()) do
+			if IsValid(v) then
+				if v:GetName() == "treatmentdoor" then v:Fire("Close") end
+				if v:GetName() == "treatmentlever" then v:Fire("PressOut") end
+			end
+		end
+	end
+end, {global=false, throttle=true})
+
+hook.Add("PlayerSay", "TreatmentRoomChat", function(ply, text, team)
+	if ply:GetLocationName() == "Treatment Room" then
+		return "i like ponies"
```

Comment:
people stuck in here won't really be able to call someone to get them out. I guess that's fine

Reply:
they should notice something's wrong if they keep saying "i love ponies" in global chat lol

Reply:
yeah, it was a relatively cheap way to get out as well. they will just have to undergo pony heaven